presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -524,16 +524,17 @@ private void toString(StringBuilder builder, int indent)
                 .collect(toImmutableList());
 
         for (Matcher matcher : matchersToPrint) {
-            builder.append(indentString(indent + 1)).append(matcher.toString()).append("\n");
+            builder.append(indentString(indent + 1, false)).append(matcher.toString()).append("\n");
         }
 
         for (PlanMatchPattern pattern : sourcePatterns) {
             pattern.toString(builder, indent + 1);
         }
     }
 
-    private static String indentString(int indent)
+    private static String indentString(int indent, boolean source)
```

Comment:
maybe instead of boolean, you could have two methods one for source and one for submatcher?